static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx

```diff
@@ -303,14 +305,14 @@ function RuleModal({
 
   const predefinedConditionsOptions = conditionCategories.map(([value, label]) => {
     // Never disable the "Add Custom Tag" option, you can add more of those
-    const disabled =
+    const disabledOption =
```

Comment:
```suggestion
    const optionDisabled =
```
just for readability wdyt?